plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -170,15 +177,16 @@ export const createPublishGitlabMergeRequestAction = (options: {
 
       try {
         const mergeRequestUrl = await api.MergeRequests.create(
-          ctx.input.projectid,
+          projectPath,
           destinationBranch,
           String(defaultBranch),
           ctx.input.title,
           { description: ctx.input.description },
         ).then((mergeRequest: { web_url: string }) => {
           return mergeRequest.web_url;
         });
-        ctx.output('projectid', ctx.input.projectid);
+        ctx.output('projectid', projectPath);
```

Comment:
Would like you another deprecated comment here? Wasn't sure about that.

Reply:
Yeah I think so. :)